lua/gitlab/actions/discussions/init.lua

```diff
@@ -202,7 +202,15 @@ M.move_to_discussion_tree = function()
   end
 
   if #d == 0 then
-    u.notify("No diagnostics for this line", vim.log.levels.WARN)
+    local warning = "No diagnostics for this line."
+    if state.settings.reviewer_settings.no_diagnostic_actions.jump then
+      warning = warning .. " Jumping to last position in discussion tree."
+      vim.api.nvim_win_set_cursor(M.split.winid, { M.last_row, M.last_column })
+      vim.api.nvim_set_current_win(M.split.winid)
+    end
+    if state.settings.reviewer_settings.no_diagnostic_actions.warn then
+      u.notify(warning, vim.log.levels.WARN)
```

Comment:
This is a pretty simple feature, and I don't think it merits two new fields in the configuration table. I'd like to keep the API surface as small as possible for users.

Let's have it be a single option called `jump_with_no_diagnostics` and if it's enabled, do not show any warning and do the jump. Otherwise (it should be the. default case) do not jump and show the warning. 

```lua
    if state.settings.reviewer_settings.jump_with_no_diagnostics then
      vim.api.nvim_win_set_cursor(M.split.winid, { M.last_row, M.last_column })
      vim.api.nvim_set_current_win(M.split.winid)
    else
      local warning = "No diagnostics for this line."
      u.notify(warning, vim.log.levels.WARN)
    end
```

Reply:
OK, fixed. Thanks for the review.